content/en/documentation/for-contributors/guidelines/pr-etiquette/_index.md

```diff
@@ -3,7 +3,46 @@ title: Pull Request Etiquette
 linkTitle: PR Etiquette
 weight: 20
 ---
-## Authors
+
+## Choosing an area to work on
+
+We welcome all contributions, after all, they are what keep the project alive and active. However, there are a few
+important things to note for a successful contribution. Choosing an area to work on is a key aspect for a
+successful pull request.
+
+As with any other open-source project, it is generally good practice to start out with some relatively low-impact
+contributions like documentation, fixing some minor bugs or generally issues with `good first issue` label on them.
```

Comment:
it could worth adding the link to the `good first issue` list page:
https://github.com/search?q=org%3Aeclipse-edc+label%3A%22good+first+issue%22+state%3Aopen&type=issues&ref=advsearch

Reply:
good point!